string name
---
string[] jointName
float64[] angle
float64 length
